CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   restart or retry.
 - New merlin example `restart_delay` that demonstrates usage of this feature.
 
+### Fixed
+- Bug that caused the merlin server to crash if given too many tasks.
```

Comment:
This isn't really a bug per se. I'd say this is a "change" to enable queue message priorities, which prioritizes "real work" over generating more work. This change enables improved scalability and server stability.